_posts/2024-03-13-zarr-sprint.markdown

```diff
@@ -0,0 +1,77 @@
+
+### Alternate backend for reading remote Zarr stores
+
+Kyle Barron worked on a prototype for an alternate store for Zarr Python using new async Python bindings to Rust's object-store project. You can see a prototype of object-store-based store implementation at [zarr-python#1661](https://github.com/zarr-developers/zarr-python/pull/1661). 
+
+## GeoZarr Specification 
+
+Throughout the sprint, the GeoZarr focus group worked on examining the interoperability of GeoZarr and different existing tooling and store support. You can see the table [here](https://github.com/zarr-developers/geozarr-spec/blob/main/geozarr-interop-table.md).
+
+One of the biggest realizations was that ArcGIS has a lot of existing support for Zarr, which is really exciting news! For other tools, there is still work to be done, especially for GeoTIFF-like data being stored in Zarr, which translates to updates needed within the GeoZarr specification. For example, there are functionality issues tied to support or lack thereof for specific compression algorithms. The GeoZarr Steering Working Group is working on providing a list of supported compressions for commonly used tools. There is also work to be done on specifying the organizational structure of GeoZarr and understanding where requirements from CF diverge from the Zarr data model. For this, we are focusing efforts on involving folks with CF expertise to guide these conversations. 
+
+If you are interested in helping out, please join the next bi-weekly GeoZarr meeting every other Wednesday at 11 EST. The next will be March 20th and you can find the invite on the Zarr calendar or join directly from [this link](https://meet.google.com/jth-rstn-fwb). Check out the notes from past meetings at the [hackmd](https://hackmd.io/@briannapagan/geozarr-spec-swg/edit).
+
+## HTTP Extension
+
+A final priority of the Zarr Sprint was to get efforts rolling on how to better visualize Zarr on the web. 
+
+Kevin Booth is the lead on this effort. Currently, he has added some sidecar files with links to reference parent, child, and root relationships in the Zarr to be able to use something like [traverzarr](https://github.com/xaviernogueira/traverzarr), the first attempt at traversing a Zarr JSON as if it were a filesystem in developed by Xavier Nogueira during the sprint, to navigate a Zarr in a manner like the Spatio-Temporal Asset Catalog (STAC). A more detailed blog post with updates on this work to come in the next week.
+
+This work continues to be worked on after the sprint. Cloud-Native Geospatial Foundation has started holding bi-weekly meetings to hack on this work. The next will be held at 12 EST on March 14th. If you would like to be involved in this, email hello@cloudnativegeo.org to be added to the meeting invite, or find the meeting link at the Zarr calendar [here](https://zarr.dev/community-calls/).
+
+---
```

Comment:
```suggestion
```
May I suggest removing this?

This separator is looking odd while reading the post. Check here: https://zarr-blog--51.org.readthedocs.build/en/51/zarr-sprint-2024/

Reply:
I'd think the reason is because the final paragraph isn't directly related to the "HTTP Extension", @MSanKeys963